docs/content/ingestion/data-formats.md

```diff
@@ -72,22 +72,22 @@ If you have nested JSON, [Druid can automatically flatten it for you](flatten-js
 
 ### CSV
 
-Since the CSV data cannot contain the column names (no header is allowed), these must be added before that data can be processed:
-
 ```json
   "parseSpec":{
     "format" : "csv",
     "timestampSpec" : {
       "column" : "timestamp"
     },
     "columns" : ["timestamp","page","language","user","unpatrolled","newPage","robot","anonymous","namespace","continent","country","region","city","added","deleted","delta"],
+    "firstRowIsHeader" : "false",
```

Comment:
Since this only works for the index task (not realtime, not hadoop) I don't think it should be in the example. Users will probably get confused when it doesn't do anything in those other modes.